### PR TITLE
refactored the muffins

### PR DESCRIPTION
**Pull request by rsimai**

### CODE REVIEW

File: recipes/desserts/chocolate_muffins_with_liquid_core.adoc [hunk 1]
```diff
@@ -18,13 +18,18 @@ Chocolate Muffins, as made by my 12y old.
...
-* 200g Sugar
-* 50g Cocoa Powder
-* 6 Eggs
-* Icing Sugar to optionally powder 
+[width="100%",cols="3,3,10,10",options="header"]
+|=========================================================
+|Metric | US    | Ingredient         | Zutat
+
+|200g   | 7oz      | Butter             | Butter
+|200g   | 7oz      | Bitter Chocolate   | Zartbitterschokolade
```
Comment (dcermak) [inline recipes/desserts/chocolate_muffins_with_liquid_core.adoc]:
```suggestion
|200g   | 7oz      | Dark Chocolate   | Zartbitterschokolade
```

File: recipes/desserts/chocolate_muffins_with_liquid_core.adoc [hunk 2]
```diff
@@ -34,7 +39,7 @@ Preparation:
 . Melt the bitter chocolate with the butter in a bain-marie and fold in the sugar-egg mass
 . Add flour and cacoa powder and stir it with the hand mixer.
 . Lube a muffin form or use paper forms, fill it with the mass
-. Bake it 15 minutes at 200°C (pre-heated), in the center
+. Bake it max. 15 minutes at 200°C/390F (pre-heated), in the center
```
Comment (dcermak) [inline recipes/desserts/chocolate_muffins_with_liquid_core.adoc]: In the "normal" mode or with convection?

File: recipes/desserts/chocolate_muffins_with_liquid_core.adoc (hunk 1 above)
Comment (rsimai) [reply recipes/desserts/chocolate_muffins_with_liquid_core.adoc]: good point, changed

File: recipes/desserts/chocolate_muffins_with_liquid_core.adoc (hunk 2 above)
Comment (rsimai) [reply recipes/desserts/chocolate_muffins_with_liquid_core.adoc]: specified

**Review by jeremy-moffitt — APPROVED** [review]
LGTM! tables are nice and clean

**Pull request merged**